webapp-django/crashstats/tokens/templates/tokens/home.html

```diff
@@ -46,7 +46,8 @@
                     <time class="ago" data-date="{{ token.created.isoformat() }}">{{ token.created.isoformat() }}</time>
                 </h4>
                 <p class="code">
-                    <code>{{ token.key[:12] }}
```

Comment:
Can you not, instead, just add HTML comments so that there is no white spaces, but still a clear indentation? Something like this: 

```html
<code>{{ token.key[:12] }}<!--
    --><span class="rest-hidden">{{ token.key[12:] }}</span><!--
    --><span class="rest-cover">...</span>
</code>
```

It seems much simpler than all these CSS tricks to me.